packages/theme-productForm/src/productForm.js

```diff
@@ -0,0 +1,93 @@
+function serializeArray(form) {
```

Comment:
I found this function - Not sure where I should put it and not sure if there's a `lodash` or similar method we could use instead